### ISSUE TITLE
Stuck in a restart loop

### ISSUE THREAD

**Issue by MeOwlei** [valnoxy/t480-oc#152]
It's just restarting over and over

**Comment by qt2409xde** [comment]
There is a fix for your problem, go to YouTube and find for EliteMacx86 channel, he will show you full steps to workaround with it

**Comment by MeOwlei** [comment]
Can you link me to the vod I can't find which one

**Comment by MeOwlei** [comment]
> There is a fix for your problem, go to YouTube and find for EliteMacx86 channel, he will show you full steps to workaround with it

Can you link me to the vod I can't find which one

**Comment by MeOwlei** [comment]
Thanks for the video link, also it turns out that I have created 2 efi partitions one of them is boot efi which loads when booting and other is just efi is that OCAT detects and thinks it's the boot efi which keeps boot efi unaffected and caused me some problems, idk what is the effective fix but for now I just copy all the file to the boot efi. And everything works now. 
I will close the issue once installation is done.

**Comment by Evanston09** [comment]
I don't really know what OCAT is but couldn't you remove the redundant EFI partition and maybe you resize another partition to take up the lost space? Good luck!

**Comment by MeOwlei** [comment]
Okay it works rn i am on Sonoma 14.5 but now my wifi is gone 

No I can't free up the redundant efi partition for some reason and it's only 200mb I will do it,but not now.

**Comment by Evanston09** [comment]
Try to use the dev branch of this repo that worked for me.

**Comment by MeOwlei** [comment]
is it because of these 3 Airportitlwm kexts which may result in conflict or maybe the kernel just ignores them as they have a different os name with them,. anyway I am gonna disable them and get new one![IMG_20240615_085349.jpg](https://github.com/valnoxy/t480-oc/assets/159135706/19c7c953-079b-4ae1-976e-f82e983e6e1c)

**Comment by MeOwlei** [comment]
> is it because of these 3 Airportitlwm kexts which may result in conflict or maybe the kernel just ignores them as they have a different os name with them,. anyway I am gonna disable them and get new one![IMG_20240615_085349.jpg](https://github.com/valnoxy/t480-oc/assets/159135706/19c7c953-079b-4ae1-976e-f82e983e6e1c)
> 

Actually it(the wifi issue) has nothing to do with any of this, it's just that Airportitlwm doesn't work with apple's secure boot being disabled which is necessary for Sonoma 14.5 . Fix : install itlwm kext with heliport

**Comment by galaxy-nomad** [comment]
How did you solve your boot loop issue? I am on Ventura but when I try to upgrade to Sonoma I am stuck in a boot loop after selecting "macos installer." I have installed RestrictEvents kext and set SecureBootModel to disabled in my BIOS and in my Config.plist.

**Comment by MeOwlei** [comment]
I don't remember exactly how i fixed it , it was a lot of trail and error , and that why try to state every new finding i got for future reference.
anyway try to go through this [video](https://www.youtube.com/watch?v=T7CDiUrWM8A) 
and also check for your boot partitions and verify that you are making changes in correct one
